### PR TITLE
fix client_golang vuln

### PR DESCRIPTION
**Pull request by fengshunli**
image

@bobbypage tested and works fine

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -27,9 +27,9 @@ require (
 	github.com/opencontainers/runc v1.1.3
 	github.com/opencontainers/runtime-spec v1.0.3-0.20210326190908-1c3f411f0417
 	github.com/pkg/errors v0.9.1
-	github.com/prometheus/client_golang v1.7.1
+	github.com/prometheus/client_golang v1.11.1
 	github.com/prometheus/client_model v0.2.0
-	github.com/prometheus/common v0.10.0
+	github.com/prometheus/common v0.26.0
```
Comment (yangjunmyfm192085) [inline go.mod:32]:
```suggestion
	github.com/prometheus/common v0.34.0
```

**Review by yangjunmyfm192085 — COMMENTED** [review]
go.sum should be modified.

**Comment by bobbypage** [comment]
Done in https://github.com/google/cadvisor/pull/3195

**Pull request closed without merging**